regression/esbmc/github_382_1/main.c

```diff
@@ -0,0 +1,8 @@
+unsigned int global_var2 = 1;
+unsigned int global_var3;
+
+int main(void) {
+  unsigned int *local_var_ptr = (unsigned int *)global_var2;
+  global_var3 = *local_var_ptr;
+  return 0;
+}
```

Comment:
I don't think this case should fail, we are dereferencing address 0x1, right?

Reply:
If you compile and run this program, it will fail due to a segmentation fault. We cannot ensure that the address 0x1 can be dereferenced here.